src/prefect/server/database/_migrations/versions/sqlite/2025_04_28_133722_7655f31c5157_remove_flow_run_notifications.py

```diff
@@ -0,0 +1,234 @@
+                )
+
+                conn.execute(
+                    sa.text("DELETE FROM flow_run_notification_policy WHERE id = :id"),
+                    {"id": row[0]},
+                )
+
+            get_logger().info(
+                f"Your {len(rows)} flow run notification policies have been migrated to automations. You can view the created automations in the Prefect UI."
+            )
+
+    op.drop_table("flow_run_notification_queue")
+    op.drop_table("flow_run_notification_policy")
+
+
+def downgrade():
+    op.create_table(
+        "flow_run_notification_queue",
+        sa.Column(
+            "id",
+            prefect.server.utilities.database.UUID(),
+            server_default=sa.text(
+                "(\n    (\n        lower(hex(randomblob(4))) \n        || '-' \n       "
+                " || lower(hex(randomblob(2))) \n        || '-4' \n        ||"
+                " substr(lower(hex(randomblob(2))),2) \n        || '-' \n        ||"
+                " substr('89ab',abs(random()) % 4 + 1, 1) \n        ||"
+                " substr(lower(hex(randomblob(2))),2) \n        || '-' \n        ||"
+                " lower(hex(randomblob(6)))\n    )\n    )"
+            ),
+            nullable=False,
+        ),
+        sa.Column(
+            "created",
+            prefect.server.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("(strftime('%Y-%m-%d %H:%M:%f000', 'now'))"),
+            nullable=False,
+        ),
+        sa.Column(
+            "updated",
+            prefect.server.utilities.database.Timestamp(timezone=True),
+            server_default=sa.text("(strftime('%Y-%m-%d %H:%M:%f000', 'now'))"),
+            nullable=False,
+        ),
+        sa.Column(
+            "flow_run_notification_policy_id",
+            prefect.server.utilities.database.UUID(),
+            nullable=False,
+        ),
+        sa.Column(
+            "flow_run_state_id",
+            prefect.server.utilities.database.UUID(),
+            nullable=False,
+        ),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_flow_run_notification_queue")),
+    )
+    op.create_index(
+        op.f("ix_flow_run_notification_queue__updated"),
+        "flow_run_notification_queue",
+        ["updated"],
+        unique=False,
+    )
+    op.create_table(
+        "flow_run_notification_policy",
+        sa.Column(
+            "id",
+            prefect.server.utilities.database.UUID(),
+            server_default=sa.text(
+                "(\n    (\n        lower(hex(randomblob(4))) \n        || '-' \n       "
+                " || lower(hex(randomblob(2))) \n        || '-4' \n        ||"
+                " substr(lower(hex(randomblob(2))),2) \n        || '-' \n        ||"
+                " substr('89ab',abs(random()) % 4 + 1, 1) \n        ||"
+                " substr(lower(hex(randomblob(2))),2) \n        || '-' \n        ||"
+                " lower(hex(randomblob(6)))\n    )\n    )"
```

Comment:
just curious for context on this part

Reply:
This is how you generate default UUIDs with SQLite, as it does not have built-in functionality for this purpose. This pattern is used whenever a new table is created.

Reply:
TIL - thanks!